CDCgov/prime-reportstream
# Upgrade com.avast.gradle.docker-compose to 0.17.4

This PR updates the com.avast.gradle.docker-compose plugin so that it works with the recent version of Docker Desktop

Test Steps:
1. Pull this branch
2. Rebuild the gradle project
3. Run `./gradlew quickRun`

Verify that you have the updated docker-desktop and if you update it, run `./gradlew quickRun` again.

## Changes
- Updates the plugin com.avast.gradle.docker-compose to 0.17.4

## Checklist

### Testing

### Process

## Linked Issues
-

## To Be Done
- 

## Specific Security-related subjects a reviewer should pay specific attention to
